.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules/*
 dist/*
 test/unit/coverage/*
+src/**/*.spec.js
```

Comment:
We need to set our set of rules and move the eslint config to own file. I want to leave that to own PR so for now skipping eslint in tests since it would be failing.

Reply:
PR here #717